### ISSUE TITLE
Model 2 and 3 Not Created

### ISSUE THREAD

**Issue by christos-efthymiou** [sokrypton/ColabFold#166]
## Expected Behavior
In the Run Prediction step, model 1 is created without issue within approximately 450 seconds. I would expect models 2 and 3 to also be accomplished within a similar timeframe.

## Current Behavior
Models 2 and 3 are not created in the Run Prediction step. I have tried several times and the colab notebook times out despite using Colab Pro. The rest of the program also does not run as a consequence

## Steps to Reproduce (for bugs)
Please make sure to reproduce the issue after a "Factory Reset" in Colab. 
If running locally ypdate you local installation `colabfold_batch` to the newest version.
Please provide your input if you can share it. 

The input sequences are (yes, the second and third are the same) KDSTGKCGPPPPIDNGDITSFPLSVYAPASSVEYQCQNLYQLEGNKRITCRNGQWSEPPKCLHPCVISREIMENYNIALRWTAKQKLYSRTGESVEFVCKRGYRLSSRSHTLRTTCWDGKLEYPTCAKR:HLIVTPSGCGEQNMIGMTPTVIAVHYLDETEQWEKFGLEKRQGALELIKKGYTQQLAFRQPSSAFAAFVKRAPSTWLTAYVVKVFSLAVNLIAIDSQVLCGAVKWLILEKQKPDGVFQEDAPVIHQEMIGGLRNNNEKDMALTAFVLISLQEAKDICEEQVNSLPGSITKAGDFLEANYMNLQRSYTVAIAGYALAQMGRLKGPLLNKFLTTAKDKNRWEDPGKQLYNVEATSYALLALLQLKDFDFVPPVVRWLNEQRYYGGGYGSTQATFMVFQALAQYQKDAPDHQELNLDVSLQLPSR:HLIVTPSGCGEQNMIGMTPTVIAVHYLDETEQWEKFGLEKRQGALELIKKGYTQQLAFRQPSSAFAAFVKRAPSTWLTAYVVKVFSLAVNLIAIDSQVLCGAVKWLILEKQKPDGVFQEDAPVIHQEMIGGLRNNNEKDMALTAFVLISLQEAKDICEEQVNSLPGSITKAGDFLEANYMNLQRSYTVAIAGYALAQMGRLKGPLLNKFLTTAKDKNRWEDPGKQLYNVEATSYALLALLQLKDFDFVPPVVRWLNEQRYYGGGYGSTQATFMVFQALAQYQKDAPDHQELNLDVSLQLPSR

use_amber is checked, and then I hit runtime - run all. 

## ColabFold Output (for bugs)
Please make sure to also post the complete ColabFold output. You can use gist.github.com for large output.

![image](https://user-images.githubusercontent.com/97161093/154851997-4b29a13b-3856-40d6-8832-f47b16522148.png)

![image](https://user-images.githubusercontent.com/97161093/154852021-67dc11c7-44d0-4862-9181-6f6e2dac457f.png)

![image](https://user-images.githubusercontent.com/97161093/154851942-0bf5933e-e907-4820-be58-ec2b7b0a5a3b.png)

## Context
Providing context helps us come up with a solution and improve our documentation for the future.

## Your Environment
Include as many relevant details about the environment you experienced the bug in.
* Git commit used
* If you run it on a local system. Please add the server specifications
* Operating system and version:

**Comment by milot-mirdita** [comment]
Does it work when amber is disabled? Maybe we have a bug with the relaxation step.

**Comment by christos-efthymiou** [comment]
Yes it does, but there is severe clashing so amber is needed to prevent this.
